Documentation/ceph-block.md

```diff
@@ -142,6 +142,150 @@ kubectl delete -n rook-ceph cephblockpools.ceph.rook.io replicapool
 kubectl delete storageclass rook-ceph-block
 ```
 
+## RBD Snapshots
```

Comment:
How about if we create a new topic for snapshots? This header should work:
```
---
title: Snapshots
weight: 3250
indent: true
---
```

Reply:
How about if we separate these doc changes into its own PR and merge it independently?

Reply:
okay, I can do it, but I wanted to keep RBD snapshot in ceph-block and cephfs snap and clone details in ceph-filesystem doc. Any suggestion?

Reply:
having a common doc with sections of RBD and CephFS is also a good idea, so that we dont want to maintain 2 docs especially when we update common thing like snapshot controller or something on common objects.

Reply:
What I like about a separate snapshot doc is that it raises visibility to the feature in the table of contents. It's a common feature to look for in storage, so raising visibility is helpful.